data_structures/Trie/Trie.java

```diff
@@ -0,0 +1,67 @@
+import java.util.* ;
+
+class Trie_Node
```

Comment:
```suggestion
class TrieNode
```

You can push changes to this branch itself. You do not need to close the PR

Reply:
Ok I will take care of it next time. I have committed your suggested change.

Reply:
Ok I will take care of it next time. I have committed your suggested change.

Reply:
Ok I will take care of it next time. I have made changes suggested by you.